docs/results.rst

```diff
@@ -0,0 +1,71 @@
+================
+Results delivery
+================
+
+All handlers can result arbitrary JSON-serializable values.
+These values are then put to the resource status under the name of the handler:
+
+.. code-block:: python
+
+    import kopf
+
+    @kopf.on.create('zalando.org', 'v1', 'kopfexampls')
+    def create_kex_1(**_):
+        return 100
+
+    @kopf.on.create('zalando.org', 'v1', 'kopfexampls')
+    def create_kex_2(uid, **_):
+        return {'r1': random.randint(0, 100), 'r2': random.randint(100, 999)}
+
+These results can be seen in the object's content:
+
+.. code-block:: console
+
+    $ kubectl get -o yaml kex kopf-example-1
+
+.. code-block:: none
+
+    ...
+    status:
+      create_kex_1: 100
+      create_kex_2:
+        r1: 66
+        r2: 666
+
+The returning results can be used to communicate between handlers through
```

Comment:
Maybe use "returned values" instead of "returning results".

Reply:
"function results" sounds better. Fixed.